code/arena.c

```diff
@@ -1364,6 +1364,24 @@ Bool ArenaHasAddr(Arena arena, Addr addr)
   return TractOfAddr(&tract, arena, addr);
 }
 
+/* ArenaAddrObject -- return base pointer of managed object */
+Res ArenaAddrObject(Addr *pReturn, Arena arena, Addr addr)
+{
+  Seg seg;
+  Pool pool;
+
+  AVER(pReturn != NULL);
+  AVERT(Arena, arena);
+
+  /* We currently assume that managed objects are inside Segments
+     and that pools where this feature is needed have Segments */
+  if (!SegOfAddr(&seg, arena, addr))
+    return ResFAIL;
```

Comment:
This is a *BAD* feature of GitHub!  It's showing my review comment on code which has changed since the comment!  That's pants.  So no, my review comment above does not apply and has been fixed.  For the record, my original comment applied to https://github.com/Ravenbrook/mps/blob/77915dc70f01a40ddeaf27b6544fb12cd5284bda/code/arena.c#L1376-L1379

Reply:
Now it's showing the correct code version.  Maybe this was a bug and they've fixed it.